frontend/server/src/Controllers/Contest.php

```diff
@@ -4041,9 +4041,12 @@ public static function apiRuns(\OmegaUp\Request $r): array {
         $result = [];
 
         foreach ($runs as $run) {
-            unset($run['run_id']);
+            $contestScore = round(floatval($run['contest_score']), 2);
+            if (!$contest->partial_score && $run['score'] != 1) {
+                $contestScore = 0;
+            }
```

Comment:
```suggestion
            if ($contest->partial_score || $run['score'] == 1) {
                $run['contest_score'] = round(floatval($run['contest_score']), 2);
            } else {
                $run['contest_score'] = 0;
            }
```

igual en todos los demás lugares donde se usa este patrón.

Reply:
Listo